_includes/config/setup-cron_2.2_how-to.md

```diff
@@ -7,25 +7,39 @@ The Magento crontab is inside `#~ MAGENTO START` and `#~ MAGENTO END` comments i
 To create the Magento crontab:
 
 1.	Log in as, or switch to, the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html).
-2.	Change to your Magento installation directory.
-3.	Enter the following command:
+1.	Change to your Magento installation directory.
+1.	Enter the following command:
 
-		php bin/magento cron:install [--force]
+    ```bash
+    bin/magento cron:install [--force]
+    ```
 
 Use `--force` to rewrite an existing Magento crontab.
 
-{:.bs-callout .bs-callout-info}
+{: .bs-callout-info }
 *	`magento cron:install` does not rewrite an existing crontab inside `#~ MAGENTO START` and `#~ MAGENTO END` comments in your crontab.
 *	`magento cron:install --force` has no effect on any cron jobs outside the Magento comments.
 
 To view the crontab, enter the following command as the Magento file system owner:
 
-	crontab -l
+```bash
+crontab -l
+```
 
 A sample follows:
 
-	#~ MAGENTO START
-	* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run | grep -v Ran jobs by schedule >> /var/www/html/magento2/var/log/magento.cron.log
-	* * * * * /usr/bin/php /var/www/html/magento2/update/cron.php >> /var/www/html/magento2/var/log/update.cron.log
-	* * * * * /usr/bin/php /var/www/html/magento2/bin/magento setup:cron:run >> /var/www/html/magento2/var/log/setup.cron.log
-	#~ MAGENTO END
+```terminal
+#~ MAGENTO START
+* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run | grep -v Ran jobs by schedule >> /var/www/html/magento2/var/log/magento.cron.log
```

Comment:
There are some things missing here, should be:
`* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run 2>&1 | grep -v "Ran jobs by schedule" >> /var/www/html/magento2/var/log/magento.cron.log`

Notice the `2>&1` and the double quotes in the grep command.